### PR TITLE
Remove zpages

### PR DESCRIPTION
**Pull request by reyang**
Remove the ZPages component:
* The total download number is very low (9.4k in the past 3+ years) https://www.nuget.org/packages/OpenTelemetry.Exporter.ZPages.
* There hasn't been any real change for years https://github.com/open-telemetry/opentelemetry-dotnet/commits/main/src/OpenTelemetry.Exporter.ZPages.
* There is no plan to make this official from the specification perspective.

### CODE REVIEW

**Review by CodeBlanch — APPROVED** [review]
LGTM

**Review by Kielek — APPROVED** [review]
AutoInstrumentation is not using this package.

Please remember to deprecate it also on nuget.org side (worth to make a bit bigger cleanup https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1100).

**Review by vishweshbankwar — APPROVED** [review]

**Comment by cijothomas** [comment]
> There is no plan to make this official from the specification perspective.

https://github.com/open-telemetry/opentelemetry-specification/blob/main/experimental/trace/zpages.md
Would this doc be updated to indicate so?

File: src/OpenTelemetry.Exporter.ZPages/README.md [hunk 1]
```diff
@@ -1,26 +0,0 @@
-# ZPages Exporter for OpenTelemetry .NET
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.ZPages/README.md:1]: maybe keep this page for sometime, with a description saying this is decommissioned. Or we think a nuget.org deprecation is fine?

**Review by cijothomas — APPROVED** [review]
Agree with this.
Maintainers may either keep a readme.md page with this info or update the nuget package to indicate its deprecaed.

File: src/OpenTelemetry.Exporter.ZPages/README.md (hunk 1 above)
Comment (reyang) [reply src/OpenTelemetry.Exporter.ZPages/README.md:1]: Both the package and the readme file have very low number of visitors so I don't feel there is a need to keep the readme.

**Review by alanwest — APPROVED** [review]

**Pull request merged**